Tests/src/TestHelpers.h

```diff
@@ -163,15 +163,14 @@ inline void SetRandSeed()
 	std::srand(static_cast<unsigned int>(std::time(nullptr)));
 }
 
-inline std::string GetUniqueHexString(int Length = 16)
+inline std::string GetUniqueString(int Length = 16)
```

Comment:
Could we get an inline explanation of the algorithm here, and how it ensures we generate unique strings per invocation?